app/models.py

```diff
@@ -1952,11 +1958,12 @@ def validates_data(self, key, data):
 
         return data
 
-    def serializable_after(self, j):
-        if 'created_at' in j:
-            j['createdAt'] = j['created_at']
+    def serializable_after(self, data):
+        if 'created_at' in data:
+            data['createdAt'] = data['created_at']
 
-        return j
+        data['signed_agreements'] = self.signed_agreements()
```

Comment:
If you make signed_agreements a property rather than a method, it should add that information to serializable without needing to touch serializable_after